en/docs/learn/configuring-oidc-federated-idp-initiated-logout.md

```diff
@@ -0,0 +1,188 @@
+# Configuring OIDC Federated IdP Initiated Logout
+
+WSO2 Identity Server (WSO2 IS) supports handling logout requests from OIDC federated identity providers. When an OIDC
+logout request is received by the OIDC federated identity provider, WSO2 IS processes the request, terminates the
+sessions and then responds to the identity provider.
+
+## Scenario
+
+The diagram below illustrates the OIDC federated identity provider initiated logout scenario. **WSO2 IS** and
+**Application 2** are configured as service providers in the **OIDC provider**. **OIDC provider** is configured as an
+identity provider, and **Application 1** is configured as a service provider in the **WSO2 IS**. When the user initiates
+the logout from **Application 2**, first the federated **OIDC provider** handles the request and propagates the logout
+request to **WSO2 IS**. After receiving the logout request from the federated identity provider, **WSO2 IS** processes
+the request and terminates the sessions and sends back a valid logout response. User is logged out from the
+**Application 1**.
+
+![oidc-fed-idp-init-logout-scenario](../assets/img/tutorials/oidc-fed-idp-init-logout-scenario.png)
+
+## Trying out the flow with WSO2 Identity Server
+
+To demonstrate the OIDC federated identity provider initiated logout, this tutorial uses two WSO2 identity servers which
+that run on ports, 9443 (Primary IS) and 9444 (Secondary IS), and two sample web applications, **Pickup-Dispatch** and
+**Pickup-Manager**. In this scenario Secondary IS acts as a federated OIDC identity provider and **Pickup-Dispatch** and
+**Pickup-Manager** act as **Application 1** and **Application 2** respectively. The following section provides a guide
+for configuring the OIDC federated identity provider initiated logout and trying it out with the sample applications.
+
+1. Configure Primary IS as a service provider in the Secondary IS
+2. Configure Secondary IS as an identity provider in the Primary IS
+3. Configure Pickup Dispatch in Primary IS
+4. Configure Pickup Manager in Secondary IS
+
+!!! note
+
+      Since there can be issues with cookies due to same hostname in both WSO2 identity servers, follow 
```

Comment:
```suggestion
      Since there can be issues with cookies when the same hostname is configured both WSO2 identity servers (primary and federated), you need to configure different hostname for both servers.. Follow
```